include/seqan/platform/platform_windows.h

```diff
@@ -71,6 +71,11 @@
 #pragma warning( disable : 4345 )
 #pragma warning( disable : 4351 )
 
+// allow multiple constructors
+#pragma warning( disable : 4521 )
```

Comment:
The nightly builds also show all the warnings for older visual studio versions
http://cdash.seqan.de/viewBuildError.php?type=1&buildid=138611
How to handle those warnings?